R/packages.R

```diff
@@ -0,0 +1,282 @@
+# Resolve package list hard dependencies
+resolve_dependencies <- function(pkgs, verbose) {
+  pkg_refs <- find.package(pkgs, lib.loc = NULL, quiet = FALSE, verbose)
+  pkg_refs <- glue::glue("installed::{pkg_refs}")
+  inst <- pkgdepends::new_pkg_installation_proposal(pkg_refs)
```

Comment:
Could this be `new_pkg_deps` as we don't need to download the packages?

```suggestion
  inst <- pkgdepends::new_pkg_deps(pkg_refs)
```

Reply:
Thanks, that worked! I also had to change `installed::` to `local::`, but I don't think it makes a difference to the result.